Sources/Authentication/AuthenticationHandler.swift

```diff
@@ -104,8 +80,11 @@ public final class AuthenticationHandler: NSObject, ObservableObject {
     /// - Returns: ``AuthenticationHandler/UserModel``
     /// - Throws: ``AuthenticationHandler/CustomError``
     public func getUser() async throws -> UserModel {
```

Comment:
```suggestion
    public func getUserOnLive() async throws -> UserModel {
```

Reply:
I am working on some changes in a new PR so to avoid conflicts I hope it is ok I merge it as it is now, and then we can fix some naming later